cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -486,13 +484,14 @@ def test_validate_default_rate_cpu_usage_cost(chargeback_costs_default, chargeba
             estimated_cpu_usage_cost = chargeback_costs_default['cpu_used_cost']
             cost_from_report = groups["CPU Used Cost"]
             cost = re.sub(r'[$,]', r'', cost_from_report)
-            assert estimated_cpu_usage_cost - 1.0 <= float(cost) \
-                <= estimated_cpu_usage_cost + 1.0, 'Estimated cost and report cost do not match'
+            assert estimated_cpu_usage_cost - DEVIATION <= float(cost) \
```

Comment:
Required: Please use brackets, like:
assert (
    estimated_cpu_usage_cost - DEVIATION <= float(cost)
    <= estimated_cpu_usage_cost + DEVIATION,
    'Estimated cost and report cost do not match'
)